Rakefile

```diff
@@ -33,7 +33,6 @@ if cucumber?
   Cucumber::Rake::Task.new(:features) do |t|
     opts = "features --format junit -o #{CUKE_RESULTS} --format pretty -x"
     opts += " --tags #{ENV['TAGS']}" if ENV['TAGS']
-    opts += " --tags ~@skip"
```

Comment:
the better fix IMO would be to remove the `skip` tag from the test that was skipped and leave this.

Reply:
for future use

Reply:
i would leave this for future use

Reply:
Done